tools/nni_cmd/nnictl_utils.py

```diff
@@ -736,3 +760,64 @@ def search_space_auto_gen(args):
         print_warning('Expected search space file \'{}\' generated, but not found.'.format(file_path))
     else:
         print_normal('Generate search space done: \'{}\'.'.format(file_path))
+
+def export_results(args):
```

Comment:
Since the command is merged to `nnictl experiment export`, is this function used?